docs/reference.rst

```diff
@@ -175,6 +176,14 @@ implement both :meth:`~AbstractStorage.load_session` and
    *httponly* -- cookie's http-only flag, :class:`bool` or ``None`` (the
    same as ``False``).
 
+   *encoder* -- session serializer.
+   A callable with the following signature: `def encode(param: Any) -> str: ...`.
+   Default is :func:`json.dumps`.
+
+   *decoder* -- session deserializer. A callable with the following signature:
+   `def decode(param: str) -> Any: ...`.
+   Default is :func:`json.loads`.
+
```

Comment:
```
.. versionadded: 2.3
    Support *encoder* and *decoder* parameters.

Reply:
````
   *encoder* -- session serializer.
   A callable with the following signature: `def encode(param: Any) -> str: ...`.
   Default is :func:`json.dumps`.

   *decoder* -- session deserializer.
   A callable with the following signature: `def decode(param: str) -> Any: ...`.
   Default is :func:`json.loads`.

   .. versionadded: 2.3
      Support *encoder* and *decoder* parameters.
````
Adding it like so doesn't seem to have any impact on the doc generation.
It seems like `versionadded` cannot just be used inside of a class description like this?